ImageBasedSimplification.ipynb

```diff
@@ -25,23 +25,6 @@
     "graph = ox.load_graphml('northeuroperails_alt.graphml')"
```

Comment:
<p>We could probably avoid self-loops by checking that from and to are different in all instances of <code>add_edge</code>  here. However, IMO doing this "postprocessing" self loop removal is a-OK too.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/laitalaj/railanalysis/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>